src/services/MQTTRoomService/MQTTRoomService.ts

```
@@ -0,0 +1,115 @@
+    [key: string]: {
+      resolve: CallableFunction;
+      reject: CallableFunction;
+    };
+  } = {};
+
+  public static async callMQTTRoomWorker<T>(
+    functionName: string,
+    args: unknown[] = [],
+    abortSignal?: AbortSignal,
+  ): Promise<T> {
+    const messageId = this.messageCounter++;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)